urllib3/util/retry.py

```diff
@@ -169,10 +184,10 @@ def get_backoff_time(self):
 
         :rtype: float
         """
-        if self._observed_errors <= 1:
+        if len(self.history) <= 1:
             return 0
 
-        backoff_value = self.backoff_factor * (2 ** (self._observed_errors - 1))
+        backoff_value = self.backoff_factor * (2 ** (len(self.history) - 1))
```

Comment:
Good point, only errors should trigger backoff.

I see the value of doing it per-host but at this point it's probably more effort than it's worth. :/